uportal-war/src/main/java/org/jasig/portal/utils/LdapQueryUtils.java

```diff
@@ -0,0 +1,34 @@
+package org.jasig.portal.utils;
+
+public class LdapQueryUtils {
+
+	/**
+	 * 
+	 * @param query
+	 * @return
+	 */
+	public static String escapeSpecialCharacters(String query) {
```

Comment:
Please fill out the method JavaDoc with a good description.  Please reference the spec; i.e. section 4 of https://docs.ldap.com/specs/rfc4515.txt.

Also I suggest renaming the method to something more clear, such as escapeLdapSearchFilterTerms to make it clear it is for the search filter as other things, such as LDAP DN strings, require different escaping.